testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -21,30 +21,22 @@
 #
 from __future__ import print_function, division, absolute_import
 
-from unittest import TestCase
-
 import MDAnalysis as mda
+import pytest
 from MDAnalysis.analysis import contacts
 from MDAnalysis.analysis.distances import distance_array
 
-from numpy.testing import (dec, assert_almost_equal, assert_equal, raises,
-                           assert_array_equal, assert_array_almost_equal)
+from numpy.testing import assert_almost_equal, assert_array_equal, assert_array_almost_equal
 import numpy as np
 
-import os
-
-from MDAnalysisTests.datafiles import (PSF,
-                                       DCD,
-                                       contacts_villin_folded,
-                                       contacts_villin_unfolded,
-                                       contacts_file, )
+from MDAnalysisTests.datafiles import PSF, DCD, contacts_villin_folded, contacts_villin_unfolded, contacts_file
 
 from MDAnalysisTests import tempdir
 
 
 def test_soft_cut_q():
     # just check some of the extremal points
-    assert_equal(contacts.soft_cut_q([0], [0]), .5)
+    assert contacts.soft_cut_q([0], [0]) == .5
     assert_almost_equal(contacts.soft_cut_q([100], [0]), 0)
     assert_almost_equal(contacts.soft_cut_q([-100], [0]), 1)
```

Comment:
@kain88-de Should I break this into two functions? One with `assert` and another with `assert_almost_equal` which would be parametrized?